app/code/Ometria/Core/view/frontend/templates/head.phtml

```diff
@@ -1,43 +1,43 @@
 <?php /** @var $this Ometria\Core\Block\Head */ ?>
 
 <?php if ($this->isUnivarEnabled()): ?>
-<script>
```

Comment:
I commented it already above:

`isUnivarEnabled()` is a bit misleading function name for this functionality

we are enabling and disabling `tracking on checkout` here not univar 😄

Reply:
@mkuzdowicz yes, I agree. This is a legacy function from the original extension code. As with all changes, where possible we leave any legacy code as is unless it is required to refactor. This is to avoid regression bugs whilst adding unrelated functionality. Given your [other comment](https://github.com/Ometria/magento2-extension/pull/32/files/fb77368404ff73b516cf73c7f33bdbf65665e43d#r717797182), I will look to rename this function when separating out the check for 'is checkout tracking allowed'